consul/serf.go

```diff
@@ -230,18 +231,29 @@ func (s *Server) maybeBootstrap() {
 		addrs = append(addrs, addr.String())
 	}
 
-	// Skip if we haven't met the minimum expect count
+	// Skip if we haven't met the minimum expect count.
 	if len(addrs) < s.config.BootstrapExpect {
 		return
 	}
 
-	// Update the peer set
-	s.logger.Printf("[INFO] consul: Attempting bootstrap with nodes: %v", addrs)
-	if err := s.raft.SetPeers(addrs).Error(); err != nil {
-		s.logger.Printf("[ERR] consul: failed to bootstrap peers: %v", err)
+	// Attempt a live bootstrap!
+	s.logger.Printf("[INFO] consul: found expected number of peers, attempting to bootstrap cluster...")
```

Comment:
I think its useful to have it log the IPs it is attempting to bootstrap with as a sanity check